lib/sqlalchemy/dialects/postgresql/asyncpg.py

```diff
@@ -237,6 +238,10 @@ class AsyncpgTime(sqltypes.Time):
     render_bind_cast = True
 
 
+class AsyncpgBit(BIT):
```

Comment:
this needs to be placed `colspecs`.

The fact that it's not and the test passes suggests that the test is not testing this part

Reply:
I think I didn't ran the tests properly before. They in fact fail now without adding `AsyncpgBit` to `colspecs`. After adding it, the test passes. So the test is doing it's job